game/scripts/npc/items/custom/item_trumps_fists.txt

```diff
@@ -68,7 +68,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_all_stats"                                 "55 75"
+        "bonus_all_stats"                                 "40 50"
```

Comment:
Adjust the stats to be the same as Skadi. Don't make the item shit.

Reply:
Judecca is way easier to apply to multiple heroes and is non committal. It should give even less stats than what I gave it but it builds out of skadi. The zeus v bat thing is a really weird thing to bring up, kinda conveniently ignoring backline applying judecca killing me in larger fights by hiding behind a massively losing 1v1 matchup for zeus

Reply:
Anyway I'll revert the damage changes and shelve it for a bit